src/server/pps/pretty/pretty.go

```diff
@@ -294,6 +296,11 @@ func PrintDetailedDatumInfo(w io.Writer, datumInfo *ppsclient.DatumInfo) {
 	tw.Flush()
 }
 
+// PrintSecretInfo pretty-prints secret info.
+func PrintSecretInfo(w io.Writer, secretInfo *ppsclient.SecretInfo) {
+	fmt.Fprintf(w, "%s\t%s\t%s\t\n", secretInfo.GetName(), secretInfo.GetType(), secretInfo.GetCreationTimestamp())
```

Comment:
Pass that creation timestamp to `pretty.Ago` to get a nicely formatted output for this. Also you can just do `secretInfo.Name` instead of `secretInfo.GetName()` I don't think going through the getter offers much benefit.